_sass/_components/lists.scss

```diff
@@ -2,7 +2,7 @@
   @include unstyled-list();
 
   li {
-    @include u-margin-bottom(2);
+    margin-bottom: units(2);
```

Comment:
@lauraGgit Out of curiosity, is there a programmatic benefit to using functions over mixins? Functions seem less wordy and more idiomatic of css, but that's just my personal preference.

Reply:
yeah, that was my thinking too -- that they are a lot more readable to someone less familiar with uswds. I also had sort of hoped that they would be slightly faster in terms of compile time, but I haven't clocked it and it seem to be that much faster